recipe/yum_requirements.txt

```diff
@@ -0,0 +1,5 @@
+java-1.8.0-openjdk-devel
+unzip
+zip
+zlib-devel
+rsync
```

Comment:
Just for my own interest, would be very curious to know what this is being used for.